lib/main_common.pm

```diff
@@ -1979,8 +1979,8 @@ sub load_security_tests_web {
     loadtest "console/wget_https";
     loadtest "console/w3m_https";
     loadtest "console/apache_ssl";
-    if (check_var('DISTRI', 'sle') && get_var('FIPS_ENABLED')) {
-        loadtest "fips/mozilla_nss/apache_nssfips";
+    loadtest "fips/mozilla_nss/apache_nssfips";
```

Comment:
`apache_nssfips` only works under fips mode

Reply:
Sorry my fault, How about changing like this simply

    if (check_var('DISTRI', 'sle') && get_var('FIPS_ENABLED')) {
       loadtest "fips/mozilla_nss/apache_nssfips";
    }
    if (is_sle('<15') && get_var('FIPS_ENABLED')) {
        loadtest "console/libmicrohttpd";
    }

Reply:
how about this
```
    if (check_var('DISTRI', 'sle') && get_var('FIPS_ENABLED')) {
        loadtest "fips/mozilla_nss/apache_nssfips";
        loadtest "console/libmicrohttpd" if is_sle('<15');
    }
```